CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+All documentation, code, communication and discussion in this repository are covered by the [W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
```

Comment:
```suggestion
All documentation, code, communication and discussion in this repository are covered by the [W3C Code of Conduct](https://www.w3.org/policies/code-of-conduct//).
```

Reply:
NB It's just "code of conduct" now and the canonical URL has changed to reflect this. The old URL redirects, but we should probably clean this up in the other repos too.